template/README.md

```diff
@@ -0,0 +1,38 @@
+---
+creation_date: 2024-05-29
```

Comment:
I'd remove all the specific details here (i.e: the date, etc.), so people can copy and paste and fill it out (or fill it with obviously junk versions)